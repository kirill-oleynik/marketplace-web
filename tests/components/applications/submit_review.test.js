const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  SubmitReview
} = require('../../../app/components/applications/submit_review');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <SubmitReview
        t={(translation) => translation}
        value={4}
        onSubmit={() => {}}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
