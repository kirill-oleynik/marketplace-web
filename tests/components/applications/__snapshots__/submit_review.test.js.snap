// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="application__submit-review"
>
  <p
    className="font-16 font-700 mb-20"
  >
    appProfile.ratings.submit
  </p>
  <div
    className="rating__stars rating__stars-0 rating__stars-open"
  >
    <i
      className="rating__stars-item icon icon-star"
      onClick={[Function]}
    />
    <i
      className="rating__stars-item icon icon-star"
      onClick={[Function]}
    />
    <i
      className="rating__stars-item icon icon-star"
      onClick={[Function]}
    />
    <i
      className="rating__stars-item icon icon-star"
      onClick={[Function]}
    />
    <i
      className="rating__stars-item icon icon-star"
      onClick={[Function]}
    />
  </div>
</div>
`;
